addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/discovery/MaxCubeBridgeDiscovery.java

```diff
@@ -65,32 +88,36 @@ public void startScan() {
 	 * @see org.eclipse.smarthome.config.discovery.AbstractDiscoveryService#
 	 * startBackgroundDiscovery()
 	 */
-
 	@Override
 	protected void startBackgroundDiscovery() {
-		discoverCube();
+		if (cubeBackgroundDiscoveryJob == null || cubeBackgroundDiscoveryJob.isCancelled()) {
+			cubeBackgroundDiscoveryJob = scheduler.scheduleAtFixedRate(cubeDiscoveryRunnable, 0, refreshInterval, TimeUnit.SECONDS);
+		}
 	}
 
 	@Override
 	public boolean isBackgroundDiscoveryEnabled() {
```

Comment:
Do not want to get off-topic, but do you think it would be good to mark the super class function final? Or do you still keep the option to overwrite it?